debian/libmiral4.symbols

```diff
@@ -419,4 +418,9 @@ libmiral.so.4 libmiral4 #MINVER#
  (c++)"miral::WaylandExtensions::zwp_virtual_keyboard_manager_v1@MIRAL_3.4" 3.4.0
  (c++)"miral::socket_fd_of(std::shared_ptr<mir::scene::Session> const&)@MIRAL_3.4" 3.4.0
  MIRAL_3.5@MIRAL_3.5 3.5.0
- (c++)"miral::WaylandExtensions::zwlr_screencopy_manager_v1@MIRAL_3.5" 3.5.0
\ No newline at end of file
+ (c++)"miral::WaylandExtensions::zwlr_screencopy_manager_v1@MIRAL_3.5" 3.5.0
+ MIRAL_3.6@MIRAL_3.6 3.6.0
+ (c++)"miral::WaylandExtensions::Context::Context()@MIRAL_3.6" 3.6.0
+ (c++)"miral::WaylandExtensions::Context::~Context()@MIRAL_3.6" 3.6.0
+ (c++)"vtable for miral::WaylandExtensions::Context@MIRAL_3.6" 3.6.0
+ (c++)"typeinfo for miral::WaylandExtensions::Context@MIRAL_3.6" 3.6.0
```

Comment:
This is wrong. `WaylandExtensions::Context` comes from MIRAL_2.5 - so it belongs in the 3.0 stanza